src/components/question-builder/components/add-answer.js

```diff
@@ -33,15 +33,19 @@ const AddAnswer = ({
         {label}
       </label>
       <div hidden={!hidden}>
-        <div className="d-inline-flex flex-column">
+        <div className="d-inline-flex flex-row">
           <input
             className="input-answer-field"
             placeholder={'Absolutely!'}
             value={answerInputText}
             onChange={e => onAnswerChange(e.target.value)}
             maxLength="18"
           />
-          <sub className="sub-text">'enter' to submit</sub>
+          <Button
+            className="button-secondary"
+            text="Submit"
+            onClick={() => (submitHandler(), onAnswerSubmit())}
```

Comment:
Why are there two submit handlers here? @maxtondelaney

Reply:
submitHandler() only sends data out and has logic to prevent empty submissions. onAnswerSubmit() clears the field and makes it hidden. I could probably put this all in one action, I'll make the modification.

Reply:
@maxtondelaney yep, should only have a single path of event firing... it make it much easier to debug later on

Reply:
<img width="472" alt="Screen Shot 2019-04-08 at 10 17 10 AM" src="https://user-images.githubusercontent.com/35947973/55743580-85494600-59e7-11e9-9b41-2136f24867c5.png">

Reply:
Got rid of the handler, instead just put the logic inline. Is this a better way to do it?

Reply:
yes, dont put logic inline like this in the renderer. 

move the answers.push to the submit handler. I dont understand would it needs to be seperate

Reply:
Here you go: 
<img width="416" alt="Screen Shot 2019-04-08 at 12 21 10 PM" src="https://user-images.githubusercontent.com/35947973/55750750-d9a8f180-59f8-11e9-9e89-5f40f076c07c.png">
<img width="565" alt="Screen Shot 2019-04-08 at 12 20 01 PM" src="https://user-images.githubusercontent.com/35947973/55750767-e3caf000-59f8-11e9-8c46-aa7bd1759cdf.png">